### ISSUE TITLE
Ctrl+Plus does nothing

### ISSUE THREAD

**Issue by Slapbox** [chrisui/react-hotkeys#47]
const keyMap = {
      'zoomIn': 'ctrl+plus',
      'zoomOut': 'ctrl+-',
    };

I tried ctrl++ also, though I was pretty sure that was wrong. Both have no effect. Any advice?

**Comment by chrisui** [comment]
Hey cheers for the report.

This will more than likely be a problem with the mousetrap library since we
just forward these key sequences there, we don't manage these ourselves.

Would you be able to search for relevant issues there and try reproducing
with the mousetrap library?

**Comment by Slapbox** [comment]
WOW that was fast!

I actually found my problem, and I thought I'd get here before anyone haha.

It's a pretty funny oversight on my part... I was using the +/= key, thinking it was PLUS by default and not EQUAL because of default browser behavior. "ctrl+plus" works fine using the numpad plus key, or adding shift in presumably. I ended up using "ctrl+="

Thanks very much for the great library and the quick response!
